posts/setting-up-a-shared-library-and-seed-job-in-jenkins-2.md

```diff
@@ -0,0 +1,189 @@
+
+1. At the root of your project, update the `Jenkinsfile` with the below code.
+```groovy
+#!/usr/bin/env groovy
+
+// Configure using microservice-pipelines and using "part2" branch
+@Library("microservice-pipelines@part2") _
+
+// Entry point into microservice-pipelines
+jenkinsJob.call()
+
+```
+### Running the `seeJob`
+All of our configuration is set up and our repositories are ready to use. We will now run our `seedJob` to create our `pipelineJob` and `multibranchPipelineJob` based on our `seed.groovy` set up.
+1. Navigate to `Dashboard` > select `seedJob` -> select `Build with Parameters` > enter `poc-micro` in `jobName` > select `Build`.
+   * **Reminder** Since we changed `seed.groovy`on our `part2` branch of `microservice-pipelines` repository, this script will require an admin approval in Jenkins.
+2. Navigate to `Dashboard` > select `Manage Jenkins` > select `In-process Script Approval` > select `Approve`.
+3. Navigate to `Dashboard` > select `seedJob` -> select `Build with Parameters` > enter `poc-micro` in `jobName` > select `Build`.
+4. Navigate to `Dashboard` > verify `poc-micro_test` and `poc-micro_deploy` jobs were created.
+   * You will need to repeat this step for all services you plan to onboard.
+![jenkins shared library configuration](hhttps://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/05/jenkins-shared-library-final-poc-micro-2.png)
+
+### Running `poc-micro_test` job
+Now you have a `poc-micro_test` job that will run every 5 minutes based on the `cron` we set up, but you can also trigger it manually.
+
+1. Navigate to `Dashboard` > select `poc-micro_test` > select `master` > select `Build Now`
+2. Under `Build History`, select the blinking blue circle (red if previous failure) > Observe `mvn test` executing in `Console Output`.
+
+### Running `poc-micro_deploy` job
+We can also observe the `poc-micro_deploy` job executing `mvn package`.
+
+1. Navigate to `Dashboard` > select `poc-micro_deploy` > select `Build Now`
+2. Under `Build History`, select the blinking blue circle (red if previous failure) > Observe `mvn package` executing in `Console Output`.
+
+## Conclusion
+During this series we set up a seed job that was used to create a `multibranchPipelineJob` and `pipelineJob` when onboarding our `poc-micro` service with our `seedJob`. Additionally, we set up our Shared Library to use `jenkinsJob.groovy` to handle the logic that determines which stages are executed depending on the currently running job (`env.JOB_NAME`). By using a combination of a `seed` job and Shared Libraries, you now have a foundation set up to onboard any number of services the same way while maintaining job specific stage ececution. This provides you the ability to streamline your CI and CD requirements.
```

Comment:
"to streamline your CI and CD requirements. " for microservices

Reply:
I updated the latest version with this suggested change